source/kaleidic/mongo_gridfs.d

```diff
@@ -0,0 +1,62 @@
+module kaleidic.mongo_gridfs;
+
+import kaleidic.mongo_standalone;
+
+// IN:     mongo            - active connection to the MongoDB instance.
+//         gridfsBucketName - full path to the GridFS parent of collections .files and .chunks.
+//         id               - the file object ID, i.e., it's "_id" value.
+// RETURN: The entire file contents in an array.
+// THROWS: File not found, or on corrupt properties.
```

Comment:
I read the ddoc page and didn't really follow it, TBH.  I've just switched to `/++ ... +/`, hopefully that's what you meant?

Reply:
yeah either `///` (single line), `/** ... */` (multi line) or `/++ ... +/` (nested multi line) means ddoc.

Just there are also standard names for the sections. In this example this is nearly ddoc, you would just change a few things: rename `IN`, `RETURN`, `THROWS` to `Params` and for the parameters instead of `-` after the argument name you use `=` + I think you need one new-line after the `Params:` line.